### ISSUE TITLE
Unable to launch API v1.1.0

### ISSUE THREAD

**Issue by issmirnov** [xBrowserSync/API#8]
Hey there,

I'm following the guide and am unable to run the latest release. Getting the following error:
```
(node:13168) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'error' of undefined
    at Server.log (/Users/vania/Data/Projects/xbrowsersync/dist/server.js:92:33)
    at Server.<anonymous> (/Users/vania/Data/Projects/xbrowsersync/dist/server.js:82:22)
    at Generator.next (<anonymous>)
    at /Users/vania/Data/Projects/xbrowsersync/dist/server.js:13:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/vania/Data/Projects/xbrowsersync/dist/server.js:9:12)
    at Server.init (/Users/vania/Data/Projects/xbrowsersync/dist/server.js:72:16)
    at Object.<anonymous> (/Users/vania/Data/Projects/xbrowsersync/dist/api.js:5:8)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
    at Function.Module.runMain (module.js:690:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:666:3
(node:13168) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:13168) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

My system information:
```
$ node --version
v8.0.0

$ mongo --version
MongoDB shell version v3.6.3
git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
OpenSSL version: OpenSSL 1.0.2n  7 Dec 2017
allocator: system
modules: none
build environment:
    distarch: x86_64
    target_arch: x86_64

$ uname
Darwin # also tried on Ubuntu 16.04
```

Error seems to be coming from [here](https://github.com/xBrowserSync/API/blob/master/dist/server.js#L92). Log file permissions are set to `777`.

Any ideas? Been looking into this in my spare time but no good suggestion yet.

**Comment by nero120** [comment]
Hi @issmirnov, I think this could be due to the service unable to create/recreate the log file on startup. Could you try setting `log.enabled` to `false` in `config/config.js` and restarting the service?

I think I need to refactor the log creation as it's quite brittle at the moment so I'll make a note to do that this week.

**Comment by issmirnov** [comment]
Hey Nero, that was exactly the problem! Set the log to false and the service started without an issue.

**Comment by nero120** [comment]
Ok good stuff @issmirnov, I'll close this when I commit a fix. 😄

**Comment by nero120** [comment]
Fixed in 9b77d77.
